### ISSUE TITLE
Version 1.5.0 is using io.opentelemetry:opentelemetry-bom with an alpha version

### ISSUE THREAD

**Issue by mamupf** [signalfx/splunk-otel-android#836]
The version 1.5.0 is using a wrong version 1.37.0-alpha for [`io.opentelemetry:opentelemetry-bom`](https://mvnrepository.com/artifact/io.opentelemetry/opentelemetry-bom) in the [BOM](https://repo1.maven.org/maven2/com/splunk/splunk-otel-android/1.5.0/splunk-otel-android-1.5.0.pom) file.

``` 
  > Could not resolve com.splunk:splunk-otel-android:1.5.0.
         > Could not parse POM com/splunk/splunk-otel-android/1.5.0/splunk-otel-android-1.5.0.pom
            > Could not find io.opentelemetry:opentelemetry-bom:1.37.0-alpha.
```

While using the alpha version the artifact id should be `opentelemetry-bom-alpha` instead of `opentelemetry-bom`.

**Comment by mamupf** [comment]
Does anyone have any issues with this?

**Comment by ber4444** [comment]
Gradle fails the build because of this
```
 > Could not parse POM https://repo.maven.apache.org/maven2/com/splunk/splunk-otel-android/1.5.0/splunk-otel-android-1.5.0.pom
               > Could not find io.opentelemetry:opentelemetry-bom:1.37.0-alpha.
```

**Comment by breedx-splk** [comment]
Sorry for the confusion around the alpha artifact/version names....Can you confirm that this is resolved for you with v1.6.0?

**Comment by mamupf** [comment]
Yes I can confirm that this is resolved in version v1.6.0. Thanks.
